package.json

```diff
@@ -31,23 +33,19 @@
   "devDependencies": {
     "@commitlint/cli": "^16.2.3",
     "@commitlint/config-conventional": "^16.0.0",
+    "@jsdevtools/host-environment": "^2.1.2",
+    "@jsdevtools/karma-config": "^3.1.7",
     "@readme/eslint-config": "^8.5.1",
     "@readme/oas-examples": "^5.0.0",
-    "alex": "^10.0.0",
```

Comment:
Removed `alex` from here because we're only running it against the root README now and it's not worth the extra heft for such a small application.